docs/coordinates/representations.rst

```diff
@@ -314,6 +314,138 @@ provides the ``base``::
   <SphericalRepresentation (lon, lat, distance) in (rad, rad, kpc)
       ( 0.0048481,  1.04719246,  1.00000294)>
 
+.. Note:: At present, the differential classes are generally meant to work with
+   first derivatives, but they do not check the units of the inputs to enforce
+   this. Passing in 2nd derivatives, e.g., acceleration values (with
+   acceleration units), will succeed, but any transformations that occur through
+   re-representation of the differential will not necessarily be correct.
```

Comment:
Unless I'm spacing here (possible, given the time), don't you need the velocities and the positions to convert the representation of an acceleration? For example, for cylindrical radius:
![image](https://user-images.githubusercontent.com/583379/27249985-ff2f7084-52f0-11e7-958e-ee64dd763578.png)